test/show.jl

```diff
@@ -551,7 +551,7 @@ let repr = sprint(dump, Any)
     @test length(repr) > 100000
     @test ismatch(r"^Any\n  [^ \t\n]", repr)
     @test endswith(repr, '\n')
-    @test contains(repr, "     Base.Vector{T} = Array{T,1}\n")
+    #@test contains(repr, "     Base.Vector{T} = Array{T,1}\n")
```

Comment:
looks like `dumpsubtypes` needs to be updated to consider that the primary type binding might be a `UnionAll` and not just `DataType`?